styles/styles.css

```diff
@@ -300,6 +321,68 @@ hr {
   text-align: center;
 }
 
+.homes {
+  background-color: #eaf0f9;
+}
+
+.homes__images {
+  gap: 16px;
+  flex-wrap: nowrap;
+  position: relative;
+  margin-bottom: 24px;
```

Comment:
mb can change to 25 px , so that there is no half-pixel

Reply:
Ok, chaneged this to 25px.